src/stream/src/from_proto/sink.rs

```diff
@@ -263,8 +263,29 @@ impl ExecutorBuilder for SinkExecutorBuilder {
                 );
 
                 let table = node.table.as_ref().unwrap().clone();
-                let input_schema = input_executor.schema();
-                let pk_info = resolve_pk_info(input_schema, &table)?;
+                let schema = if let Some(extra_partition_col_idx) =
```

Comment:
Changing the schema may bypass the log store schema validation, but if the input is still in its original schema, we will get error, or write incorrect content to log store. How does this PR fix this? Is the extra partition column already removed from the input?

Reply:
No, the extra column is still kept in the input. Interesting. There is no error and the log store works normally, we can select the log store internal table as well.

Reply:
Just checked the code. When the schema of row does not match the schema of the serializer, there won't be error when encoding the row and writing to storage. Error only happens when trying to decode the encoded byte slice to row. When the amount of data is less than the log store in memory buffer, rows are only written to storage, but won't be read, and we will read the original stream chunk. So I guess this is why there is no error currently.